src/Microsoft.Azure.SignalR.Common/Auth/AadApplicationOptions.cs

```diff
@@ -4,11 +4,10 @@
 using System;
 using System.Security.Cryptography.X509Certificates;
 using System.Threading.Tasks;
-using Microsoft.Azure.SignalR.Common.Auth;
 
 namespace Microsoft.Azure.SignalR
 {
-    public class AadApplicationOptions : AuthOptions, ITokenBasedAuthOptions
+    public class AadApplicationOptions : AuthOptions, IAadTokenGenerator
```

Comment:
this is a public class, so this is a breaking change? does `1.5.0` have this already?